resources/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
-INSTALL(FILES srs.db qgis.db symbology-style.xml spatialite.db customization.xml 2to3migration.txt
+INSTALL(FILES srs.db
+              qgis.db
+              symbology-style.xml
+              spatialite.db
+              customization.xml
+              2to3migration.txt
+              qgis_global_settings.ini
         DESTINATION ${QGIS_DATA_DIR}/resources)
-INSTALL(FILES qgis_global_settings.ini
```

Comment:
@elpaso I see this was originally going to ${QGIS_DATA_DIR} but given that means it's in a different place to source vs install location it means we have to extra path resolve to get it. I think it makes sense to just keep in resources. 

@jef-n if this PR goes though I will follow up with package changes to reflect this.

Reply:
@nathanw2 done in 17039ff1954